velox/functions/prestosql/InPredicate.cpp

```diff
@@ -140,6 +140,9 @@ class InPredicate : public exec::VectorFunction {
       case TypeKind::TINYINT:
         filter = createBigintValuesFilter<int8_t>(inputArgs);
         break;
+      case TypeKind::BOOLEAN:
+        filter = createBigintValuesFilter<bool>(inputArgs);
```

Comment:
It is hacky to create BIGINT filter for boolean values. Perhaps, add a comment to avoid confusing future readers.